lib/screens/departures/departure.ex

```diff
@@ -385,8 +385,12 @@ defmodule Screens.Departures.Departure do
     {"include", Enum.join(relationships, ",")}
   end
 
+  defp format_query_param({:date, %DateTime{} = date}) do
+    {"filter[date]", Date.to_iso8601(date)}
```

Comment:
Oops, I had you add a bug! 😱 (Though it may have also been there with the earlier Timex version)

You need to convert to local time before formatting to a date timestamp. `Date.to_iso8601` called on a DateTime simply grabs the year/month/day fields without regard for timezone, so this will produce the wrong date if called on a DateTime that's late at night in local time:
```ex
iex> ten_pm_on_the_twenty_sixth_eastern = ~U[2023-03-27T02:00:00Z]
iex> Date.to_iso8601(ten_pm_on_the_twenty_sixth_eastern)
"2023-03-27"
```

So this should be
```suggestion
    dt_in_local_time = DateTime.shift_zone(date, "America/New_York")
    {"filter[date]", Date.to_iso8601(dt_in_local_time)}
```